wp-content/themes/pub/wporg-learn-2024/js/course-outline.js

```diff
@@ -0,0 +1,38 @@
+/* global wporgCourseOutlineData */
+
```

Comment:
Sensei doesn't allow customization of the status icons, and they're hardcoded in HTML. Here can only use JavaScript to insert the desired icons.